prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.ts

```diff
@@ -221,9 +238,9 @@ export class CollegeCertificationFormComponent implements OnInit {
     return this.practices.filter(p => p.collegePractices.map(cl => cl.collegeCode).includes(collegeCode));
   }
 
-  private checkLicenceCodeValidatedByPharmaNet(licenceCode: number): boolean {
+  private getPrescriberIdType(licenceCode: number): PrescriberIdType | undefined {
```

Comment:
It's a bit weird to have a return type that can be explicitly `undefined`.  I would add an appropriate default of `null` using null coalesce - `...pop() ?? null`